backend/src/main/kotlin/app/ehrenamtskarte/backend/application/webservice/schema/create/BlueCardApplication.kt

```diff
@@ -0,0 +1,51 @@
+package app.ehrenamtskarte.backend.application.webservice.schema.create
+
+import app.ehrenamtskarte.backend.application.webservice.schema.view.JsonField
+import app.ehrenamtskarte.backend.application.webservice.schema.view.Type
+import app.ehrenamtskarte.backend.application.webservice.utils.JsonFieldSerializable
+
+data class BlueCardApplication(
+    val personalData: PersonalData,
+    val applicationType: ApplicationType,
+    val entitlement: BlueCardEntitlement,
+    val hasAcceptedPrivacyPolicy: Boolean,
+    val givenInformationIsCorrectAndComplete: Boolean
+) : JsonFieldSerializable {
+    override fun toJsonField(): JsonField {
```

Comment:
Looks strange to me. Is this the format in which application are stored in order to be displayed later? What turns out to be easier by doing so?

Reply:
We want to ensure, that we can always display all applications in a sensible manner even if we change the form for applications.
Therefore we have to persist e.g. also "translations" / label strings s.t. we can always display the values together with their corresponding labels.

Reply:
I think most ways are much work. So what you have done is a bit hacky but also a bit nice :D